src/Education/IEducation.cs

```diff
@@ -69,6 +69,12 @@ public void ShowRuleHelp(SonarCompositeRuleId ruleId)
         {
             var ruleInfo = ruleMetadataProvider.GetRuleInfo(ruleId);
 
+            if (ruleInfo == null)
+            {
+                logger.WriteLine(Resources.Education_NoRuleInfo, ruleId.ErrorListErrorCode);
+                return;
+            }
```

Comment:
Just erring on the side of caution. This shouldn't happen in practice, but if it does we end with a null-ref exception on the UI thread.